src/extension.ts

```diff
@@ -130,6 +140,13 @@ function registerVSCodeCommands(context: ExtensionContext) {
     WelcomeWebview.createOrShow(context);
   }));
 
+  /**
+   * Command for opening CodeLens URL
+   */
+  context.subscriptions.push(commands.registerCommand(VSCodeCommands.QUARKUS_OPEN_URL, (url) => {
```

Comment:
Why you need that?

Reply:
Looks like it is some old code from before, I will remove it